### DISCUSSION TITLE
Prestashop Basic Edition

### DISCUSSION THREAD

**Discussion by darkf3d3** [PrestaShop/PrestaShop#32491] [Q&A] [answered]
Going to download prestashop from prestashop.com I saw new layout of the site including a new logo, and also that now PS it's called Prestashop Basic Edition.
What does it mean exaclty? That you will realease a free basic version and a paid version with more features?

**Comment by kpodemski** [comment, marked as answer]
Hello @darfk3d3 

We discuss open-source project things there, you can download every version of the PrestaShop from the [releases page](https://github.com/PrestaShop/PrestaShop/releases).

Regarding the "Basic Edition", I think there's a chance for PrestaShop SA to, at some point, release a version with more features, I think that's the case already in France, where there's a paid version with hosting, and a few extra modules included :-)

But, as I wrote above, it's not really a topic for the open-source project because nothing changes here.

**Comment by matks** [comment]
Hi there!

As mentioned before, the PrestaShop Basic Edition is a product of PrestaShop company and is not in the scope of this open source project so it's out-of-scope.

That being said I can tell you what I know 😉 : if today you download current Basic Edition from prestashop.com ; what you will get is a regular PrestaShop 8.0.4 bundled together with some modules from PrestaShop company, such as https://github.com/prestashopcorp/ps_mbo which allows users of the Basic Edition to connect their shop with addons.prestashop.com and install & upgrade the modules they have bought on Addons. There is more modules in Basic Edition.

So PrestaShop Basic Edition is a ZIP which contains the regular PrestaShop open source software, augmented with a choice of modules by the company. Something you can do too: this is called a [Distribution](https://devdocs.prestashop-project.org/8/basics/deployment/distribution/#why-create-a-distribution). This is similar to what happens with Linux: you can download Ubuntu, Archlinux, Kubuntu, Fedora... all of them are based on Linux software, but each of them bring specific additions.
